frontend/src/app/events/events-info/events-info.component.html

```diff
@@ -68,7 +68,7 @@ <h6 *ngIf="dataHistory.length > 0">Datenveränderungen</h6>
             </ng-container>
             <ng-container matColumnDef="bearbeiter">
                 <mat-header-cell *matHeaderCellDef> Bearbeiter </mat-header-cell>
-                <mat-cell *matCellDef="let element"> {{element.userName}}
+                <mat-cell *matCellDef="let element"> {{element.user.firstName + " " + element.user.lastName}}
```

Comment:
ich würde hier mit einem Fragezeichenoperator arbeiten. 
Um sicher zu gehen, dass die Property auch gesetzt ist.
Falls nicht gesetzt ist gibt es einen Fehler in der Console

Reply:
erledigt